GoogleChromeLabs/ps-analysis-tool
# Feature: Add known breakages section in aggregated report

## Description
This PR adds code to add the Known Breakages section to the sitemap landing page and the report.


## Relevant Technical Choices



## Testing Instructions
- Run the CLI on a sitemap where known breakages occur.
- Open the dashboard HTML file and navigate to the bottom.
- The Known breakages section should be present at the bottom.
- Now download the report and check the HTML files, it should contain known breakages section.


Tested On: https://sitemap.superintegratedapp.com/sitemaps/sitemap.xml



## Additional Information:



## Screenshot/Screencast
<img width="1470" alt="Screenshot 2024-07-10 at 13 27 22" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/5d967c4e-1528-45ce-998d-1f07789ce191">
<img width="1470" alt="Screenshot 2024-07-10 at 13 27 23" src="https://github.com/user-attachments/assets/0f3c61c2-7256-431b-969d-ebd5a455b0b7">



---

## Checklist



- ~This code is covered by unit tests to verify that it works as intended.~



Fixes #752
